README.md

```diff
@@ -315,51 +341,6 @@ import "github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
 import "github.com/Azure/azure-sdk-for-go/profiles/preview/compute/mgmt/compute"
 ```
 
-## Inspecting and Debugging
```

Comment:
why remove this part ?

Reply:
This might have been due to a bad merge, will check...

Reply:
This is due to some changes that were made directly in master thus they aren't in latest.  Will fix then update this PR.